### PR TITLE
adding configureUnlock

### PR DESCRIPTION
**Pull request by julien51**
# Description

Version 7 introduces a new way to configure the unlock contract so I needed to make this version specific rather than "global".

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [X] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-js/src/v13/configureUnlock.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/**
+ * Creates a lock on behalf of the user, using version v0
```
Comment (akeem) [inline unlock-js/src/v13/configureUnlock.js]: Does this comment reflect the code below? Is there a reason why we would be using v0 here rather than v13?

Comment (julien51) [reply unlock-js/src/v13/configureUnlock.js]: ha! Wrong comments. Let me fix.

**Comment by julien51** [comment]
So the duplication is actually kind of necessary for now because we may/will get rid of some versions eventually and that's easier if each version is full self-contained.

**Comment by julien51** [comment]
> Could definitely be an issue if the behavior is expected to change from version to version

Ha, yes I should have clarified that indeed v7 is different which is why this is moving from the shared "trunk" to the local versions.

**Pull request merged**